packages/node-core/src/integrations/http/index.ts

```diff
@@ -13,6 +13,14 @@ interface HttpOptions {
    */
   breadcrumbs?: boolean;
 
+  /**
+   * Whether to create spans for requests or not.
+   * As of now, creates spans for incoming requests, but not outgoing requests.
+   *
+   * @default `true`
```

Comment:
l: also needs to be changed here, I guess? (see other comment)

```suggestion
   * @default `true` unless `skipOpenTelemetrySetup` is set to `true` in `Sentry.init` options
```